tests/scalability/generate_brac_data/README.md

```diff
@@ -0,0 +1,46 @@
+# Brac Config Data Generator Script (first draft)
+
+## Introduction
+
+This script is to generate Brac config test data.
+
+Initially it only creates places, persons, users and reports (pregnancy, assessment and assessment follow up). Taking into account hierarchies.
+
+Changes where made in /test/factories/ and /test/scalability/
+Basically added new folders:
+/test/factories/brac and /test/scalability/generate_brac_data
+
+## Requirements
+
+- NodeJS
+- a target medic instance with brac config loaded
+
+
+## Steps to execute it
+1. Clone cht-core repo.
+2. Run npm ci, npm install and npm install @faker-js/faker --save-dev in cht-core root directory.
```

Comment:
These dependencies should be added in the root package, so that `npm ci` would suffice.
Alternatively, you could just install it in this folder, and require running `npm ci` here as well.